alembic-autogenerate.py

```diff
@@ -2,26 +2,26 @@
 
 #
 # How to use this script:
-# 1. Load the LSST environment and setup sdm_schemas and felis.
-#        source loadLSST.bash
-#        setup felis
-#        setup -r /path/to/sdm_schemas
+# 1. Install Felis and sdm_schemas, set environment variables:
+#        pip install lsst-felis testing.postgresql alembic sqlalchemy pyyaml
+#        git clone https://github.com/lsst/sdm-schemas
+#        cd sdm_schemas
+#        export SDM_SCHEMAS_DIR=`pwd```
 # 2. From the root of the consdb git repo, invoke the script. Supply a
 #    revision message as the command line argument:
-#        python alembic-autogenerate.py this is my revision message "\n" \
-#            the message can span multiple lines "\n" \
-#            if desired
+#        python alembic-autogenerate.py DM-12345
```

Comment:
I don't know that we have "officially" determined that ticket names will be used in migrations. Maybe it needs to be discussed in the meeting first?

Reply:
Did we officially decide something? it wasn't super clear to me from the recording